util/version/Cargo.toml

```diff
@@ -1,10 +1,23 @@
+# NOTE This file is used by the auto-updater service.
+# Make sure to update the service if it's moved or the structure is changed.
 [package]
 name = "parity-version"
 # NOTE: this value is used for Parity version string (via env CARGO_PKG_VERSION)
 version = "1.10.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 build = "build.rs"
 
+[package.metadata]
+# This versions track. Should be changed to `stable` or `beta` when on respective branches.
+# Used by auto-updater and for Parity version string.
+track = "nightly"
+
+# Latest supported fork blocks for various networks. Used ONLY by auto-updater.
+[package.metadata.forks]
```

Comment:
Where are these values used?

Reply:
I guess nowhere in parity... `Used ONLY by auto-updater` so https://github.com/paritytech/push-release . am I right @tomusdrw ?